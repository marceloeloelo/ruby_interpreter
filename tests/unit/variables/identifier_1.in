identifier

identifier1

identifier_1009

identifier_1_ident
