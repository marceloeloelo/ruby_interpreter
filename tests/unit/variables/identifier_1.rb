identifier = 1

identifier1 = 1

identifier_1009 = 1

identifier_1_ident = 1

@var = 1
