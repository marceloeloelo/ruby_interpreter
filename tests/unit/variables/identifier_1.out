NameError: undefined local variable or method `identifier' for main:Object
